src/preact-render-spy.js

```diff
@@ -99,15 +136,40 @@ class SpyWrapper {
     return new FindWrapper(this, [this.vdomMap.get('root')], selector);
   }
 
-  render(vdom) {
+  _createSpy(vdom, depth) {
+    return createSpy;
+  }
+
+  __render(vdom) {
```

Comment:
since we don't really change anything about this method between the two wrappers, perhaps we should just call it `render` here and on the shallow wrapper?

`new DeepWrapper().render(jsx)` vs `new ShallowWrapper.render(jsx)`